scripts/install/install_hil.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+user=`whoami`
+
+sudo mkdir /opt/hil
+sudo chown $user:$user /opt/hil
+sudo chmod 775 /opt/hil
+
+git clone http://github.com/cci-moc/hil /opt/hil
+
+pushd /opt/hil/
+
+virtualenv .venv
+source .venv/bin/activate
+pip install -e .
+
+cp examples/haas.cfg.dev-no-hardware ./haas.cfg
+
+export HAAS_ENDPOINT=http://127.0.0.1:5000
+export HAAS_USERNAME=haas
+export HAAS_PASSWORD=secret
+
+# Start Haas services
+haas-admin db create
+haas serve 5000&
+haas serve_networks&
+
+sleep 5
+
+# Create Haas projects
+haas project_create bmi_infra
+
+### Setup HaaS mock node
+haas node_register bmi_node mock moch-hostname mock-username mock-password
+haas project_connect_node bmi_infra bmi_node
+
+### Setup Mock Node that BMI deploys to
+qemu-img create -f qcow2 /opt/bmi_virtual_node.qcow2 8G
+sudo virt-install -n BMIVM  --description "Test BMI VM" --os-type=Linux --os-variant=rhel6 --ram=2048 --vcpus=2 --disk path=/opt/bmi_virtual_node.qcow2,bus=virtio,size=10 --pxe --network bridge:virbr0 &
```

Comment:
Yeah, this would be a nice to have feature but I think it needs more work. We shall remove it and add it later.